Dockerfile.test

```diff
@@ -13,7 +13,7 @@ sh -s -- -b $(go env GOPATH)/bin v1.24.0
 
 RUN         go get -u \
 github.com/onsi/ginkgo/ginkgo@v1.12.2 \
-golang.org/x/tools/cmd/goimports@v0.0.0-20200520220537-cf2d1e09c845 \
+golang.org/x/tools/cmd/goimports@v0.0.0-20200616195046-dc31b401abb5 \
```

Comment:
If only we didn't squash commits we could write a long commit message about how this broke openshift/client-go dependencies at compile time so if it happens again we wouldn't have to remember all of that pain :trollface:

Reply:
For future us:

This was causing compile time errors because an old version was being used when building the application:
```
08:30:07 /go/pkg/mod/github.com/openshift/client-go@v0.0.0-20190923180330-3b6373338c9b/config/clientset/versioned/typed/config/v1/apiserver.go:55:5: not enough arguments in call to c.client.Get().Resource("apiservers").Name(name).VersionedParams(&options, scheme.ParameterCodec).Do
08:30:07 	have ()
08:30:07 	want (context.Context)
```